src/ecs/resources/input.rs

```diff
@@ -22,14 +22,52 @@ impl<'a> Iterator for PressedKeysIterator<'a> {
     }
 }
 
+/// Iterator that iterates through all currently pressed down mouse buttons
+pub struct PressedMouseButtonsIterator<'a> {
```

Comment:
Yes please, that would be a good idea. Sorry for asking for you to do it, I should have caught it before merging. :persevere: